data/json/items/generic/dining_kitchen.json

```
@@ -292,14 +292,22 @@
   },
   {
     "id": "bowl_plastic",
-    "type": "CONTAINER",
+    "type": "GENERIC",
     "category": "other",
     "name": { "str": "plastic bowl" },
     "symbol": "u",
     "description": "A plastic bowl with a convenient sealing lid.  Holds 750 ml of liquid.",
     "copy-from": "base_plastic_dish",
     "volume": "750 ml",
-    "container_data": { "contains": "750 ml", "seals": true, "watertight": true },
+    "pocket_data": [
+      {
+        "pocket_type": "CONTAINER",
+        "watertight": false,
```

Why does this stop being watertight? Shouldn't that be reflected in description at least?

---

possibly read it wrong,
```suggestion
        "watertight": true,
```